src/main/java/org/triplea/client/launch/LaunchScreen.java

```diff
@@ -0,0 +1,654 @@
+package org.triplea.client.launch;
```

Comment:
I'd suggest we keep the Swing client in distinct packages from the JFX client.  The JFX client is currently rooted at `org.triplea.client.ui.javafx`, so following that convention would produce something like `org.triplea.client.ui.swing.launch` for this package.  Everything's still relatively new, so if you want to tweak the fully-qualified name, I don't think that will be an issue.  Regardless of what convention is chosen, the Swing and JFX client packages should probably be siblings and not directly overlap.

(We still have an open issue, #1945, for package naming conventions under `org.triplea.*` if you think this discussion needs to move outside this PR.)